src/dev/build/tasks/nodejs/node_download_info.ts

```diff
@@ -22,7 +22,8 @@ export function getNodeDownloadInfo(config: Config, platform: Platform) {
     } else {
       variants = ['glibc-217'];
     }
-    if (platform.isServerless()) variants.push('pointer-compression');
+    // disabled, see https://github.com/nodejs/node/issues/54531
+    // if (platform.isServerless()) variants.push('pointer-compression');
```

Comment:
We never moved this to production.  For now this also disables the test pipeline and we'll revisit bringing it back or removing the source separately.